src/Stratis.Features.FederatedPeg.IntegrationTests/CrossChainTransferTests.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Stratis.Features.FederatedPeg.IntegrationTests
 {
-    public class NodeSetupTests
+    public class CrossChainTransferTests
     {
         [Fact(Skip = "Unstable. https://dev.azure.com/StratisProject/StratisBitcoinFullNode/_build/results?buildId=14582&view=ms.vss-test-web.build-test-results-tab")]
```

Comment:
![image](https://user-images.githubusercontent.com/29645989/59826637-07f06900-937a-11e9-8242-ebbd8cc75b6b.png)

Reply:
Yeah this test failed:

`NodeServer_Enabled_When_ConnectNode_Args_Specified_And_Listen_Specified [FAIL]`